spec/helpers/application_helper_spec.rb

```diff
@@ -29,15 +29,21 @@
 
   describe 'body_classes' do
     context 'with a body class string from a controller' do
-      before do
-        without_partial_double_verification do
-          allow(helper).to receive_messages(body_class_string: 'modal-layout compose-standalone', current_theme: 'default', current_account: Fabricate(:account))
-        end
-      end
+      before { helper.extend controller_helpers }
```

Comment:
Sounds good, but I haven't found that to be a common pattern and I'm slightly worried `extend`ing the `helper` object could cause issues down the line.

Reply:
I see the fear there - we don't want to modify what's available from the helper in other areas outside this context ... but I'm pretty sure this extension only exists within this spec context, and there's an implicit teardown when other specs run ... I verified this just by doing a `helper.methods.size` check within this spec and within some other random spec, and confirming that this spec number was 3 higher than the other one, no matter what order they ran in. If we don't trust that, we could add a more explicit teardown which removes the added methods (I don't think you can "unextend" a module, but we could do them individually)